scan-engine/src/main/java/de/tillhub/scanengine/google/ui/GoogleScanningActivity.kt

```diff
@@ -34,25 +35,29 @@ internal class GoogleScanningActivity : AppCompatActivity() {
 
     private lateinit var camera: Camera
 
-    private val requestPermissionLauncher = registerForActivityResult(
-        ActivityResultContracts.RequestPermission()
-    ) { isGranted ->
-        if (isGranted) {
-            bindCamera()
-        } else {
-            Toast.makeText(
-                baseContext,
-                R.string.error_permission_not_granted,
-                Toast.LENGTH_SHORT
-            ).show()
-            binding.requestPermission.isVisible = true
+    @VisibleForTesting
+    val isCameraInitialized get() = this::camera.isInitialized
+
+    private val requestPermissionLauncher by lazy {
+        registerForActivityResult(
+            ActivityResultContracts.RequestPermission(),
+            activityResultRegistry
+        ) { isGranted ->
+            if (isGranted) {
+                bindCamera()
+            } else {
+                Snackbar.make(binding.root, R.string.error_permission_not_granted, Snackbar.LENGTH_SHORT).show()
+                binding.requestPermission.isVisible = true
+            }
         }
     }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
         setContentView(binding.root)
+        setSupportActionBar(binding.toolbar)
+        supportActionBar?.title = EMPTY
```

Comment:
There is better way supportActionBar.setDisplayShowTitleEnabled(false) and you can delete EMPTY